src/main/java/com/github/monster860/fastdmm/FastDMM.java

```diff
@@ -805,4 +870,59 @@ private void loop() {
 			Display.sync(60);
 		}
 	}
+
+	private String getAppData() {
```

Comment:
@PJB3005 There are tons of applications that simply drop a `.applicationname` folder in the users home folder, doing this works cross-platform without the need for a shitty function to handle all these edge cases.